nixos/modules/tasks/filesystems/bcachefs.nix

```diff
@@ -42,7 +42,10 @@ in
 {
   config = mkIf (elem "bcachefs" config.boot.supportedFilesystems) (mkMerge [
     {
-      system.fsPackages = [ pkgs.bcachefs-tools ];
+      # we don't want bachefs in fsPackages for systemd-initrd because we will
+      # to provide the unwrapped version of mount.bcachefs via extraBin into scope.
```

Comment:
I think this comment needs rewording. (I failed to parse it.)

Reply:
rewritten.